src/coreclr/gc/gc.cpp

```diff
@@ -14293,7 +14293,11 @@ HRESULT gc_heap::initialize_gc (size_t soh_segment_size,
 #endif //MULTIPLE_HEAPS
 
 #ifdef MULTIPLE_HEAPS
+#ifdef DYNAMIC_HEAP_COUNT
+    yp_spin_count_unit = 10;
```

Comment:
Would it be better to set the low initial spin counts unconditionally for all flavors?

Reply:
Just took a look at more data and concluded that for WKS and SVR, we aren't exhibiting the case where we don't have the Yield Processor Factor before the first GC that was the cause of the regression. We can possibly consider setting an default agnostic of flavor after doing more analysis on other benchmarks.